bastion_current_ip.py

```diff
@@ -0,0 +1,27 @@
+# obtain current public ip, create a variables.tfvars file 
+# with this ip as the user_ip (along with other static data). 
+# Run terraform shell script to apply the instance
+
+import os,subprocess
+
+myip = subprocess.check_output(["dig","+short","myip.opendns.com","@resolver1.opendns.com"])
+
+myip = myip.rstrip()
+ssh_key_name = "WallOfSound"
```

Comment:
You may want to consider using a command line option for these variables so you can change these.